cdap-cli/src/main/java/co/cask/cdap/cli/command/GetStreamEventsCommand.java

```diff
@@ -86,12 +86,12 @@ public String getPattern() {
   @Override
   public String getDescription() {
     return "Gets events from " + Fragment.of(Article.A, ElementType.STREAM.getName()) + ". " +
```

Comment:
Unrelated to your PR, but since you're modifying this, it would be worth changing this to use ``String.format`` like most other descriptions for consistency and readability.